### DISCUSSION TITLE
Different objects contributing to the same loitering timer

### DISCUSSION THREAD

**Discussion by R3DW1N3** [blakeblackshear/frigate#14420] [Ask A Question]
I have a camera pointed at a road with a zone configured to track cars with loitering time 20 seconds. The issue that I'm seeing is that if two different cars enter that zone in the span of 20 seconds, it's triggering an alert. 

1) Car comes into frame of camera - spends 10 seconds in zone before leaving frame.
2) 5 seconds pass with no cars in frame.
3) Second car comes into frame - spends 10 seconds in zone.
4) Alert is triggered.

Anything I can do to prevent this from happening?

**Comment by NickM-27** [comment]
it is impossible for multiple objects to share the same loitering timer because the loitering timer is a property of the object itself.

**Reply by R3DW1N3** [reply]
Right, that's my understanding as well. It's not possible that it's counting both cars as the same object for some reason? There's a solid 5 seconds between the two cars being on-camera.

**Reply by NickM-27** [reply]
the only way would be if the cars drove right past each other and the tracker swapped the IDs

**Reply by R3DW1N3** [reply]
It's probably not that, then. I dunno, I'll do some more tweaking and see if I can figure out what's going on. Thanks.
